tests/testcases/mod.rs

```diff
@@ -14,6 +14,98 @@ fn get_url(base: &str, path: &str) -> String {
     format!("{}{}", base, path)
 }
 
+// Cannot run on Spin controller as `spin up` does not take a variables flag
+#[allow(dead_code)]
+pub async fn config_variables_works(controller: &dyn Controller) {
```

Comment:
Hi Kate, I think this is the same test we talked about. we should probably move this test to cloud tests repo